common/utils/src/main/resources/error/error-conditions.json

```diff
@@ -4731,6 +4731,11 @@
           "<treeNode>"
         ]
       },
+      "UNSUPPORTED_CORRELATED_EXPRESSION_IN_JOIN_CONDITION": {
+        "message" : [
+          "Correlated subqueries in the join predicate cannot reference both join inputs."
+        ]
```

Comment:
"subqueryExpression" : "exists(x.x2, y.y2, (z.z2 = x.x2), (z.z2 = y.y2))"

Where is this `subqueryExpression` defined?

Reply:
It's passed as a parameter when the error is thrown, here https://github.com/apache/spark/blob/master/sql/catalyst/src/main/scala/org/apache/spark/sql/catalyst/optimizer/subquery.scala#L216-L217

Reply:
Does it get shown in the error message? If not would be good to add it similar to <treeNode> in similar error messages

Reply:
I thought `subqueryExpression` should be defined as a part of error message so you need to pass that in?

Reply:
Fixed! Weirdly enough, the error message in the golden files didn't change.